src/views/admin/PersonDetailsView.vue

```diff
@@ -136,6 +136,28 @@
                 {{ personStore.currentPerson.person.referrer }}
               </v-col>
             </v-row>
+            <!-- Kopers-Nr -->
+            <v-row>
+              <v-col cols="1"></v-col>
+              <v-col
+                class="text-right"
+                md="2"
+                sm="3"
+                cols="5"
+              >
+                <span class="subtitle-2"> {{ $t('person.kopersnr') }}: </span>
+              </v-col>
+              <v-col
+                cols="auto"
+                data-testid="person-kopersnr"
+              >
+                {{
+                  personStore.currentPerson.person.personalnummer
+                    ? personStore.currentPerson.person.personalnummer
+                    : '---'
```

Comment:
Funktioniert hier auch das?
```suggestion
                  personStore.currentPerson.person.personalnummer ?? '---'
```